lecli/logset/commands.py

```diff
@@ -0,0 +1,97 @@
+"""
+Logset commands module
+"""
+import sys
+import json
+import click
+
+from lecli.logset import api
+
+
+@click.command()
+@click.option('-n', '--name', type=click.STRING, help="Name of new log")
+@click.option('-f', '--filename', type=click.Path(exists=True, dir_okay=False),
+              help="Full or relative path to file containing JSON log object")
+def createlogset(name=None, filename=None):
+    """
+    Create a logset with the provided name and details.
+    This method will use the JSON file first if both name and file are provided
+    """
+    if filename is not None:
```

Comment:
We are not checking the case of both name and file being provided. Should we not check that?